app/src/main/java/com/android/shelfLife/ui/overview/FoodDisplayComponents.kt

```diff
@@ -134,7 +134,7 @@ fun FoodItemCard(
       when (foodItem.foodFacts.quantity.unit) {
         FoodUnit.GRAM -> "g"
         FoodUnit.ML -> "ml"
-        FoodUnit.COUNT -> "in stock"
+        FoodUnit.COUNT -> " in stock"
```

Comment:
## Good fix
There was a padding issue between the count and the text. Maybe this should be part of a UI fix/refactor issue, but I see no problem with it being corrected in this PR since you are refactoring many little things